react-catalog/my-app/src/Catalog.tsx

```diff
@@ -11,7 +12,7 @@ export function Catalog() {
         const catalog = await readCatalog();
         setProducts(catalog);
       } catch (error) {
-        console.error(error);
+        alert(error);
```

Comment:
This way the user sees it, but it's also good to log it. Sometimes the log message contains information not available in the alert. Or the user may dismiss the alert and then call you for help. If you can say, "Do x-y-z to open the console, and read the error message to me" then you may be able to figure out what happened, and fix it!